packages/contracts/contracts/L1/KromaPortal.sol

```diff
@@ -327,10 +347,9 @@ contract KromaPortal is Initializable, ResourceMetering, ISemver {
      *
      * @param _tx Withdrawal transaction to finalize.
      */
-    function finalizeWithdrawalTransaction(Types.WithdrawalTransaction memory _tx)
-        external
```

Comment:
Could you please explain why the data location of the _tx parameter in the external function of Solidity is set to memory instead of calldata? I understand that calldata is preferred for read-only parameters in external functions because it consumes less gas.

Reply:
Yeah I think you're right and it's better to use `calldata` in the respect of gas usage. But Optimism uses `memory` here, so I think it's better to remain it to reduce the diff.